src/environment/interval_adjustor.py

```diff
@@ -0,0 +1,21 @@
+from abc import ABCMeta, abstractmethod
+from time import perf_counter
+
+
+class IntervalAdjustor(metaclass=ABCMeta):
```

Comment:
基底から抽象クラスを定義する場合はこちらの書き方が主流ですが、本質的には同じ機能と意味を持つので修正しなくて大丈夫です。
```suggestion
from abc import ABC, abstractmethod
from time import perf_counter


class IntervalAdjustor(ABC):
```

Reply:
myxyさんの実装がこうだったんで、見慣れないなりにやった感じすね